### DISCUSSION TITLE
Need to add the infinity as a datasource in Grafana using Terragrunt.

### DISCUSSION THREAD

**Discussion by manimcabit** [grafana/grafana-infinity-datasource#693] [Q&A]
we have added the infinity plugin as Datasource in Grafana, the same task we would like to automate via terraform/terragrunt. 
To achieve that, do we have any blogs or official documentation.

**Comment by yesoreyeram** [comment]
Refer installation and provisioning section of the docs https://sriramajeyam.com/grafana-infinity-datasource/docs/provisioning
